maps/hazards.go

```diff
@@ -590,6 +640,59 @@ func (m RiverAndBridgesHazardsMap) UpdateBoard(lastBoardState *rules.BoardState,
 	return StandardMap{}.UpdateBoard(lastBoardState, settings, editor)
 }
 
```

Comment:
The starting places don't seem to be symmetrical - what's the thinking behind which ones are left out? I don't think we necessarily need support for 16 snakes (i.e. 4 in each quadrant) but just wondering how they've been chosen.

Reply:
The positions came from the Python source that Chris shared with me in [the linear ticket](https://linear.app/battlesnake/issue/DEV-1412/fix-hz-rivers-bridges-map-in-rules-to-include-snake-starting-positions).

Reply:
@robbles this was an artifact of this map originally being used for showdowns with up to 16 snakes which was covered in the original logic that Torben was porting. 

I think adding some randomness is fine as long as we are trying to distribute the snakes between the quadrants relatively evenly.  Ultimately I think beyond 4 snakes it is hard to do a purely "fair" distribution of snakes on this map so i think we can do whatever will make for a more interesting game

Reply:
Ah yeah I can see how it's important to ensure even weighting of quadrants. We couldn't naively shuffle the start points, because you could end up with some pretty unbalanced quadrants.

I think that a simple solution is to shuffle the snakes that get placed. This would add randomness and preserve the way the current start positions order keeps the quadrants evenly distributed.

Reply:
@robbles what about the solution I just pushed where I shuffle the order that the snakes get placed? This way it's not entirely predictable, but we still have the quadrants distributed.

I could go further as well. This approach adds some randomness, but I could introduce more by shuffling the start points in a way that still doesn't allow one quadrant to become overloaded.

Reply:
I feel like a quadrant shuffling approach could work well

Reply:
Sorry, I hadn't refreshed and my comment was missing your context. I'll take a look now.

Reply:
I'm not seeing a change that shuffles the order? Did that commit get pushed?

Reply:
I think shuffling the snakes, then placing in order based on the start positions being evenly distributed makes sense though. That sounds like a great solution.